app/org/sagebionetworks/bridge/play/controllers/ActivityEventController.java

```diff
@@ -0,0 +1,30 @@
+package org.sagebionetworks.bridge.play.controllers;
+
+import org.joda.time.DateTime;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import play.mvc.Result;
+
+import org.sagebionetworks.bridge.json.DateUtils;
+import org.sagebionetworks.bridge.models.accounts.UserSession;
+import org.sagebionetworks.bridge.services.ActivityEventService;
+
+/**
+ * Created by jyliu on 6/5/2017.
+ */
+@Controller
+public class ActivityEventController extends BaseController {
+    @Autowired
+    private ActivityEventService activityEventService;
+
+    public Result createActivityEvent(String eventKey, String timestamp) {
+        UserSession session = getAuthenticatedSession();
+
+        DateTime eventTime = DateUtils.getDateTimeOrDefault(timestamp, null);
```

Comment:
Is it nice to allow the server to set the timestamp? If so, we'd set the default value.